packages/pretty-format/src/collections.ts

```diff
@@ -51,6 +52,13 @@ export function printIteratorEntries(
     const indentationNext = indentation + config.indent;
 
     while (!current.done) {
+      result += indentationNext;
+
+      if (width++ === config.maxWidth) {
+        result += '...';
```

Comment:
```suggestion
        result += '…';
```

I think that makes sense?

Reply:
I'll update as a separate commit to include the test changes as well